modules/express/src/typedRoutes/api/v2/lightningInitWallet.ts

```diff
@@ -0,0 +1,39 @@
+import * as t from 'io-ts';
+import { httpRoute, httpRequest, optional } from '@api-ts/io-ts-http';
+import { BitgoExpressError } from '../../schemas/error';
+
+/**
+ * Path parameters for initializing a Lightning wallet
+ * @property {string} coin - A lightning coin name (e.g, lnbtc).
+ * @property {string} walletId - The ID of the wallet.
+ */
+export const LightningInitWalletParams = {
+  coin: t.string,
```

Comment:
Should we be more specific here if this is only supporting lightning?

lnbtc or tlnbtc? Maybe check with the btc team regarding this route.